### PR TITLE
fix: initial selected value not getting displayed

### LINKED ISSUE #14999: [Bug]: Default selected value of border radius or box shadow is not visible in Table column property
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

The default selected value of border-radius or box-shadow for Table widget's column property is not visible.
It should be shown which item is selected out of the available options.

Screenshot 2022-07-05 at 10 40 42 AM
 

### Steps To Reproduce

1. Add Table widget
2. Goto action column's property and take a look at border-radius & box-shadow

### Public Sample App

_No response_

### Version

1.7.6

### PR DESCRIPTION
**Pull request by aswathkk**
## Description

Since we are using COMPUTE_VALUE customJS control for Table columns, the evaluated value sent to the property controls becomes an array and the controls fail to render the initial selected item.
The fix is to send only one value out of the array. This logic might fail if users are using some bindings that can generate different values inside an array. But, we are anyway disabling the JS toggle for those scenarios.

Fixes #14999

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

- Manual

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/utils/PropertyControlFactory.tsx [hunk 1]
```diff
@@ -37,19 +38,24 @@ class PropertyControlFactory {
...
     let controlBuilder;
+    let evaluatedValue = controlData.evaluatedValue;
 
     if (preferEditor) {
       controlBuilder = customEditor
         ? this.controlMap.get(customEditor)
         : this.controlMap.get("CODE_EDITOR");
     } else {
+      if (customEditor === "COMPUTE_VALUE" && isArray(evaluatedValue)) {
+        evaluatedValue = evaluatedValue[0];
```
Comment (riodeuno) [inline app/client/src/utils/PropertyControlFactory.tsx:49]: Would it make more sense to use `if(uniq(evaluatedValue).length === 1)` condition before assigning?

Comment (riodeuno) [reply app/client/src/utils/PropertyControlFactory.tsx:49]: I am assuming the case in which all values are the same, and in which all values are not the same. If I understand the solution correctly, we allow the JS toggle if all the values in the array are the same. Is this correct?

Comment (aswathkk) [reply app/client/src/utils/PropertyControlFactory.tsx:49]: We disable the JS toggle if there are any bindings except the binding of the default theme. So, what I was saying is that on line number 49, the value of `evaluatedValue` will always be an array where all the values are the same.

**Review by riodeuno — APPROVED** [review]

**Comment by Sripriya93** [comment]
Have verified Table widget "Action" column's Button property and style, moving to done

**Pull request merged**